bitnami/openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -150,6 +150,19 @@ ldap_validate() {
     [[ "$error_code" -eq 0 ]] || exit "$error_code"
 }
 
+########################
+# Check if OpenLDAP is ready to answer
+# Globals:
+#   LDAP_PID_FILE
+# Arguments:
+#   None
+# Returns:
+#   Whether slapd is running
+#########################
+is_ldap_answering() {
```

Comment:
Hi @tatankat, thanks for your contribution!

To maintain consistency, we use to name with the `ready` keyword this kind of functions. I would rename it to `is_ldap_ready`.

Reply:
You're welcome. I have changed the name and updated the comments.